### PR TITLE
INTS-113 & INTS-114 - Only add relationships if we haven't seen the key before

### PR DESCRIPTION
**Pull request by mknoedel**

### CODE REVIEW

File: src/util/safeAddRelationship.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { JobState, Relationship } from '@jupiterone/integration-sdk-core';
+
+export async function safeAddRelationship(
+  jobState: JobState,
+  relationshipKeys: Set<string>,
+  relationship?: Relationship,
+) {
+  if (relationship && !relationshipKeys.has(relationship._key)) {
+    await jobState.addRelationship(relationship);
+    relationshipKeys.add(String(relationship._key));
```
Comment (austinkelleher) [inline src/util/safeAddRelationship.ts:10]: Do we have to use `String` here?

**Review by austinkelleher — COMMENTED** [review]
Is the `_key` actually unique enough? Under what circumstance would there be a duplicate key?

File: src/util/safeAddRelationship.ts (hunk 1 above)
Comment (mknoedel) [reply src/util/safeAddRelationship.ts:10]: We do not. I'll remove that.

**Comment by mknoedel** [comment]
I'm going to do this in a better way.

**Pull request closed without merging**